use_case_guide/real_time_dashboards.rst

```diff
@@ -71,11 +63,24 @@ data across your cluster after adding new worker nodes.
 
   Citus Cloud uses `streaming replication <https://www.postgresql.org/docs/current/static/warm-standby.html>`_ to achieve high availability and thus maintaining shard replicas would be redundant. In any production environment where streaming replication is unavailable, you should set ``citus.shard_replication_factor`` to 2 or higher for fault tolerance.
 
-With this, the system is ready to accept data and serve queries! We've provided `a data
-ingest script
-<https://github.com/citusdata/realtime-dashboards-resources/blob/master/ingest_example_data.sql>`_
```

Comment:
Not sure why we used to link to an external file. I inlined the sql in the guide instead.